pyvista/plotting/charts.py

```diff
@@ -1,12 +1,14 @@
 """Module containing pyvista wrappers for the vtk Charts API."""
-
 from functools import wraps
 import inspect
 import itertools
 import re
 from typing import Dict, Optional, Sequence
 import weakref
 
+from matplotlib.backends.backend_agg import FigureCanvasAgg
+import matplotlib.figure  # noqa
```

Comment:
@dcbr why was this import necessary originally? Do we still need it if matplotlib is a hard dependency? If it tests something indirectly, can we leave a comment for what it does?

Reply:
Yea the `import matplotlib.figure` was originally a check to see if it was available or not. So if we make it a hard dependency this line can definitely be removed. I think it could have been removed a while ago already, as it is no longer wrapped within a `try-catch` blok either.